.github/workflows/hugo.yml

```diff
@@ -16,6 +16,32 @@ jobs:
         uses: peaceiris/actions-hugo@v2
         with:
           version: 0.83.1
+      - name: Build search index
+        run: |
+          LATEST_RELEASE=$(curl -L -s -H 'Accept: application/json' https://github.com/arranf/HugoToJSON/releases/latest)
+          LATEST_VERSION=$(echo $LATEST_RELEASE | sed -e 's/.*"tag_name":"\([^"]*\)".*/\1/')
+          ARTIFACT_URL="https://github.com/arranf/HugoToJSON/releases/download/$LATEST_VERSION/hugo_to_json"
```

Comment:
I'd rather pin a version than always grab the latest here.

Reply:
Pinned latest version (0.3.7) as of now.